addons/repair/models/repair.py

```diff
@@ -581,7 +581,8 @@ class RepairLine(models.Model):
     product_id = fields.Many2one('product.product', 'Product', required=True)
     invoiced = fields.Boolean('Invoiced', copy=False, readonly=True)
     price_unit = fields.Float('Unit Price', required=True, digits='Product Price')
-    price_subtotal = fields.Float('Subtotal', compute='_compute_price_subtotal', store=True, digits=0)
+    price_subtotal = fields.Float('Subtotal', compute='_compute_price_total_and_subtotal', store=True, digits=0)
+    price_total = fields.Float('Total', compute='_compute_price_total_and_subtotal', store=False, compute_sudo=True, digits=0)
```

Comment:
computed fields are non store by default so you can skip this parameter. Same for compute sudo

Reply:
compute_sudo is False by default for non stored fields, but I removed the store parameter.